client/lib/abtest/active-tests.js

```diff
@@ -1,5 +1,15 @@
 /** @format */
 export default {
+	multiyearSubscriptions: {
+		datestamp: '20180417',
+		variations: {
+			show: 50,
```

Comment:
Do you mind starting with 10/90 split instead of 50/50?

Reply:
Done, thanks!